src/odin/server/status/mod.rs

```diff
@@ -0,0 +1,79 @@
+mod bepinex_info;
+mod jobs_info;
+
+use crate::constants::{AUTO_BACKUP_JOB, AUTO_UPDATE_JOB};
+use crate::utils::environment::fetch_var;
+use a2s::info::Info;
+use a2s::A2SClient;
+use bepinex_info::BepInExInfo;
+use jobs_info::JobInfo;
+use log::{debug, error};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct ServerInfo {
+  pub name: String,
+  pub version: String,
+  pub players: u8,
+  pub max_players: u8,
+  pub map: String,
+  pub online: bool,
+  pub bepinex: BepInExInfo,
+  pub jobs: Vec<JobInfo>,
+}
+
+impl ServerInfo {
+  pub fn new(address: &str) -> ServerInfo {
+    let parsed_address = address.replace('"', "");
```

Comment:
Yup, this shouldn't be needed if you don't do debug formatting to begin with

Reply:
I am keeping the replace in there in case someone uses the env variable for `ADDRESS` :P